xla/tools/hlo_opt/BUILD

```diff
@@ -59,7 +62,6 @@ cc_library(
         "//xla/service:hlo_graph_dumper",
         "//xla/service:platform_util",
         "//xla/service/gpu:executable_proto_cc",
-        "//xla/stream_executor/cuda:cuda_platform_id",
```

Comment:
This dependency is actually needed, we have a header include in gpu_opt.cc.

Reply:
Won't having dependency on //xla/stream_executor/cuda:cuda_platform be enough?

Reply:
Yes, that works as well. Turns out the include was not used.